cmd/kubeadm/app/componentconfigs/kubelet.go

```diff
@@ -231,3 +239,18 @@ func isServiceActive(name string) (bool, error) {
 	}
 	return initSystem.ServiceIsActive(name), nil
 }
+
+// MutateCgroupDriver can be called to set the KubeletConfiguration cgroup driver to systemd
+// Currently this cannot be as part of Default() because the function is called for
+// upgrades too, which can break existing nodes after a kubelet restart.
+func MutateCgroupDriver(cfg *kubeadmapi.ClusterConfiguration) {
+	kubeletConfig := cfg.ComponentConfigs[KubeletGroup]
+	raw := kubeletConfig.Get().(*kubeletconfig.KubeletConfiguration)
```

Comment:
Show we make a type assertion?

Reply:
probably better to panic here.
it means that something is setting the kublet config key to a bogus structure.